File: MODUL 1/SOAL 4/PRAK104-2610817310016-MuhammadIjlal.c
#include <stdio.h>

int main() {
    int price_a = 400000;
    int price_b = 350000;

    int discount_a = price_a - (price_a * 13 / 100);
    int discount_b = price_b - (price_b * 21 / 100);

    printf("Harga sepatu A adalah %d\n", price_a);
    printf("Harga sepatu B adalah %d\n", price_b);
    printf("Sepatu A mendapat diskon 13%% sehingga harganya menjadi %d\n", discount_a);
    printf("Sepatu A mendapat diskon 21%% sehingga harganya menjadi %d\n", discount_b);

    return 0;
}

File: MODUL 1/SOAL 4/PRAK104-2610817310016-MuhammadIjlal.py
price_a = 400000
price_b = 350000

discount_a = price_a - (price_a * 13 // 100)
discount_b = price_b - (price_b * 21 // 100)

print(f"Harga sepatu A adalah {price_a}")
print(f"Harga sepatu B adalah {price_b}")
print(f"Sepatu A mendapat diskon 13% sehingga harganya menjadi {discount_a}")
print(f"Sepatu B mendapat diskon 21% sehingga harganya menjadi {discount_b}")